diff -ruN teapop-0.27.orig/config/configure.in teapop-0.27/config/configure.in
--- teapop-0.27.orig/config/configure.in	Tue Oct 31 18:48:34 2000
+++ teapop-0.27/config/configure.in	Sun Dec 17 00:33:14 2000
@@ -132,7 +132,7 @@
 	fi
 	test -f $db_pgsql/include/postgres.h && pgsql_cflags=$db_pgsql/include
 	test -f $db_pgsql/pgsql/include/postgres.h && pgsql_cflags=$db_pgsql/pgsql/include
-	test -f $db_pgsql/include/pgsql/postgres.h && pgsql_cflags=$db_pgsqlinclude/pgsql
+	test -f $db_pgsql/include/pgsql/postgres.h && pgsql_cflags=$db_pgsql/include/pgsql
 	if test "$pgsql_cflags" = ""; then
 		AC_MSG_ERROR([can't find PostgreSQL include directory])
 	fi
@@ -160,7 +160,7 @@
 	fi
 	test -f $db_mysql/include/mysql.h && mysql_cflags=$db_mysql/include
 	test -f $db_mysql/mysql/include/mysql.h && mysql_cflags=$db_mysql/mysql/include
-	test -f $db_mysql/include/mysql/mysql.h && mysql_cflags=$db_mysqlinclude/mysql
+	test -f $db_mysql/include/mysql/mysql.h && mysql_cflags=$db_mysql/include/mysql
 	if test "$mysql_cflags" = ""; then
 		AC_MSG_ERROR([can't find MySQL include directory])
 	fi
